test/header/CMakeLists.txt

```diff
@@ -12,11 +12,10 @@ include (../seqan3-test.cmake)
 
 option (SEQAN3_FULL_HEADER_TEST "Test seqan3 headers as well as the headers of external libraries" OFF)
 
-# We compile each header twice in separate compilation units. Each alone is
-# sufficient to test that the header is functional, but both are needed to check
-# for link errors, which can happen if the header accidentally defines a
-# variable, e.g. a global or class static member. Furthermore this tests that
-# header guards are working by including the same header twice.
+# We compile each header twice in separate compilation units, where the second compilation omits the header guard (check
```

Comment:
This will make the header tests take even longer right? Not that I'm against this check but just  thinking of the long CI times.

Reply:
Mhmm, but the header CI tests are among the first that finish because of ccache. At least I did not need to wait for header tests but rather the OS machines being spun up. Here it says it was successfull in 2 minutes, which isn't too bad.